releasenotes/notes/rcm-fix-disable-f45787a4e6576f53.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Validate DD_REMOTE_CONFIGURATION_ENABLED environment variable is false before enable Remote Config worker and 
+    register Remote Config products
```

Comment:
I don't think this is clear to a user what the issue was.

Can we state what the impact was and what the fix is more directly?

```
Ensure DD_REMOTE_CONFIGURATION_ENABLED environment variable disables remote config
```
?